trino/trino.go

```diff
@@ -576,11 +586,14 @@ func (st *driverStmt) ExecContext(ctx context.Context, args []driver.NamedValue)
 		queryID:      sr.ID,
 		nextURI:      sr.NextURI,
 		rowsAffected: sr.UpdateCount,
+		statsCh:      st.statsCh,
+		doneCh:       st.doneCh,
 	}
 	// consume all results, if there are any
 	for err == nil {
 		err = rows.fetch(true)
 	}
+	st.doneCh <- struct{}{}
```

Comment:
You should close this channel here.

Reply:
added `close(st.doneCh)` on line after this.

Reply:
Just close it without sending anything. You can't receive an empty struct anyway, so closing is equivalent and it can be done only once.